Python/Product/VSInterpreters/Interpreter/CondaEnvironmentManager.cs

```diff
@@ -50,15 +52,21 @@ sealed class CondaEnvironmentManager : ICondaEnvironmentManager, IDisposable {
         public static CondaEnvironmentManager Create(IServiceProvider serviceProvider) {
             var condaPath = CondaUtils.GetRootCondaExecutablePath(serviceProvider);
             if (!string.IsNullOrEmpty(condaPath)) {
-                return new CondaEnvironmentManager(condaPath);
+                return Create(condaPath);
             }
 
             return null;
         }
 
+        public static CondaEnvironmentManager Create(string condaPath) {
```

Comment:
More descriptive name please. "CreateCondaEnvironmentManager"?

Reply:
`Create` is standard naming convention in .NET for this scenario.

`CondaEnvironmentManager.CreateCondaEnvironmentManager` would be long and redundant.